README.md

```diff
@@ -29,16 +29,16 @@ Or install it yourself as:
 
 `Avromatic` supports the following configuration:
```

Comment:
This configuration is only required if you're using a schema registry right? If so, we should probably make some changes to the wording of this section.